swift-ci/sdks/android/build-docker

```diff
@@ -70,6 +59,5 @@ $DOCKER run -i --rm \
             --products-dir "/products" \
             --host-toolchain "/usr/local/swift" \
             --build-compiler "${BUILD_COMPILER}" \
-            --android-api "${ANDROID_API}" \
```

Comment:
Why remove this?

Reply:
The android-api is just set in the default in build.sh; we never override it from `build-docker` or `build-local`, so I removed it so where is just a single place where we define the default.

Reply:
I thought the idea was that people using these scripts could pass in `ANDROID_API` and build the SDK for other API levels too.